lesson28/test_regestration.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+from selenium.webdriver import Chrome
+
+from lesson28.garage_page import GaragePage
+from lesson28.main_page import MainPage
+from lesson28.registration_page import RegistrationPage
+
+
+@pytest.fixture
+def driver():
+    driver = Chrome()
+    driver.maximize_window()
+    yield driver
+    driver.quit()
+
+
+def test_registration(driver):
+    main_page = MainPage(driver)
+    registration_page = RegistrationPage(driver)
+    garage_page = GaragePage(driver)
+
+    main_page.open_main_page("https://guest:welcome2qauto@qauto2.forstudy.space")
+    main_page.sign_up_button()
```

Comment:
it is better to name methods as the action they do, like
click_sign_up_button
input_signup_name
...